decidim-core/app/packs/src/decidim/sw/loader.js

```diff
@@ -1,8 +1,15 @@
 // check if the browser supports serviceWorker at all
 window.addEventListener("load", async () => {
   if ("serviceWorker" in navigator) {
-    // eslint-disable-next-line no-unused-vars
-    const registration = await navigator.serviceWorker.register("/sw.js", { scope: "/" });
+    await navigator.serviceWorker.register("/sw.js", { scope: "/" });
+
+    const mandatoryElements = document.querySelector(".js-sw-mandatory");
+    const isOperaMini = (navigator.userAgent.indexOf("OPR") > -1) && (navigator.userAgent.indexOf("Mobile") > -1);
+
+
+    if (mandatoryElements && !isOperaMini) {
+      mandatoryElements.style.display = "block";
```

Comment:
Could we instead use the `hide` class and then add it here when using opera mini?